deps.ffmpeg/30-libvpx.zsh

```diff
@@ -81,9 +81,14 @@ config() {
         args+=(--target="${arch}-darwin20-gcc")
       }
       ;;
-    windows-x*)
+    windows-*)
       cross_prefix="${target_config[cross_prefix]}-w64-mingw32-"
-      args+=(--enable-runtime-cpu-detect --target="${target_config[gcc_target]}")
+      args+=(--target="${target_config[gcc_target]}")
```

Comment:
Use separate `windows-arm64` branch, there is no need to check the target again when you could've matched the target in the `case` statement.

Reply:
done